tables/gr-bb.ctb

```diff
@@ -1,7 +1,7 @@
-# liblouis: Unofficial table used by some for classical and biblical Greek.
+# liblouis: Unofficial table used by some for studying classical and biblical Greek.
 #
 # -----------
-#-display-name: Ancient Greek braille
+#-display-name: Greek braille used by some scholars
```

Comment:
What's important I think is the fact that the table used by non-Greek speaking. That it is for studying is less relevant. Its useful information but I would drop it from the "display-name" metadata because this is what is presented to users and those descriptions should be as concise and accurate as possible.